rust/template/ovsdb/lib.rs

```diff
@@ -227,11 +227,17 @@ fn record_from_val(v: Value) -> Result<Record, String> {
     }
 }
 
+type NameTranslator<'a> = &'a dyn Fn(&'a str) -> Result<&'static str, String>;
+
```

Comment:
This is very limiting, it should be a generic instead of forcing dynamic dispatch
```suggestion
```

Reply:
Can you please write the suggested replacement? It took me 2 days to figure out even this one.

Reply:
Unfortunately not, trait aliases aren't stable yet